doc/languages.rst

```diff
@@ -949,6 +951,19 @@ to view the docstring of the syntax checker.  Likewise, you may use
          A list of additional library directories. Relative paths are relative
          to the buffer being checked.
 
+   .. syntax-checker:: rust-cargo-clippy
```

Comment:
If I understand correctly this checker is a superset of `rust-cargo`, isn't it?  

In this case please add it to the previous `.. syntax-checker::` directive with the other rust checkers, so that we don't need to document each option twice.  You can move the `.. note::` up as well and just change it to 

```
The `rust-cargo-clippy` syntax checker requires …
```